example.py

```diff
@@ -670,6 +670,7 @@ def core(self,slothy):
         slothy.config.variable_size=True
         slothy.config.inputs_are_outputs = True
         slothy.optimize(start="start", end="end")
+        slothy.fusion_region("start", "end", ssa=False)
```

Comment:
You need to put the fusion_region before the optimize, otherwise this does not help SLOTHY find a better solution.